<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_device">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Device</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>VID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>PID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="HexSpinBox" name="sb_VID"/>
        </item>
        <item row="1" column="1">
         <widget class="HexSpinBox" name="sb_PID"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_open">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_read">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_read_eeprom">
        <property name="text">
         <string>read_eeprom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_write_eeprom">
        <property name="text">
         <string>write_eeprom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_eeprom_decode">
        <property name="text">
         <string>eeprom_decode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_write">
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_test">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexSpinBox</class>
   <extends>QSpinBox</extends>
   <header>hexspinbox.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sb_VID</tabstop>
  <tabstop>sb_PID</tabstop>
  <tabstop>btn_open</tabstop>
  <tabstop>btn_read</tabstop>
  <tabstop>btn_read_eeprom</tabstop>
  <tabstop>btn_write_eeprom</tabstop>
  <tabstop>btn_eeprom_decode</tabstop>
  <tabstop>btn_write</tabstop>
  <tabstop>btn_close</tabstop>
  <tabstop>btn_test</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
